### DISCUSSION TITLE
No audio output

### DISCUSSION THREAD

**Discussion by laluxx** [ppy/osu#20574] [Q&A] [answered]
### Type

Game behaviour

### Bug description

i just installed osu-lazer on an arch based system whit the yay helper
the package i installed is (aur/osu-lazer-git 2022.605.1-1 (+41 0.34))
but when i open the game there is no audio output 
also a bunch of error pops out

### Version

lazer 2022.923.1 

### Logs

[database.log](https://github.com/ppy/osu/files/9706573/database.log)
[input.log](https://github.com/ppy/osu/files/9706574/input.log)
[network.log](https://github.com/ppy/osu/files/9706575/network.log)
[performance.log](https://github.com/ppy/osu/files/9706576/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9706577/runtime.log)

**Comment by huupoke** [comment, marked as answer]
https://github.com/ppy/osu/issues/11991#issuecomment-794864483
